src/Context/ImplicitContextKeyedInterface.php

```diff
@@ -26,8 +26,8 @@ interface ImplicitContextKeyedInterface
     public function activate(): ScopeInterface;
 
     /**
-     * Returns a new {@see Context} created by setting `$this` into the provided [@see Context}.
-     * It is suggested to use {@see Context::withContextValue()} instead of using this method directly.
+     * Returns a new {@see ContextInterface} created by setting `$this` into the provided [@see ContextInterface}.
+     * It is suggested to use {@see ContextInterface::withContextValue()} instead of using this method directly.
```

Comment:
What is the reason for this suggestion? Using this method feels more fluent.
```php
$spanBuilder
    ->setParent($parent)
    ->startSpan()
    ->storeInContext($parent)
    ->activate()
```

Reply:
It's before my time, I think it's ok to remove the comment.